build/CMakeCache.txt

```diff
@@ -0,0 +1,38 @@
+# This is the CMakeCache file.
```

Comment:
please don't commit the `build` directory.
And please add it to the `.gitignore` file so that it doesn't happen accidentally.